src/app/lib/config/services/yoruba.js

```diff
@@ -2,16 +2,19 @@ import { yoruba as brandSVG } from '@bbc/psammead-assets/svgs';
 import { latin } from '@bbc/gel-foundations/scripts';
 
 const yoruba = {
+  product: 'BBC News',
   brandName: 'BBC News Yorùbá',
   locale: 'yo',
   service: 'yoruba',
   serviceName: 'Yoruba',
+  serviceLocalizedName: 'Yorùbá',
```

Comment:
@DenisHernandez can we have this `product `and `serviceLocalizedName` added to `persian` service as well?